src/Symfony/Bridge/Twig/Mime/NotificationEmail.php

```diff
@@ -166,7 +180,9 @@ public function getPreparedHeaders(): Headers
 
         $importance = $this->context['importance'] ?? self::IMPORTANCE_LOW;
         $this->priority($this->determinePriority($importance));
-        $headers->setHeaderBody('Text', 'Subject', sprintf('[%s] %s', strtoupper($importance), $this->getSubject()));
+        if ($this->context['importance']) {
```

Comment:
I think this may not be set.

Should we modify the `$importance = ` line above instead? default it to `IMPORTANCE_LOW` if it is NOT set, but allow it to use null if it IS set to null?

Then, down here, we would just `if ($importance)`.

If we did that, I think `determinePriority()` would need to be tweaked to allow a `?string $importance` argument.

Reply:
I think it's always set, as the default is set in the `$context` property to `IMPORTANCE_LOW`. I'm not sure why the `??` is there are line 181, probably some left-over legacy check.

Checking `determinePriority` with `IMPORTANCE_LOW` is fine, it'll result in normal priority (so a non-prioritized email).